locales/de.json

```diff
@@ -60,7 +60,7 @@
     "config": {
         "checking-config": "Konfiguration wird überprüft...",
         "done-with-checking": "Konfiguration erfolgreich überprüft.",
-        "creating-file": "Konfiguration %m/%f exestiert aktuell nicht, wird aber gleich erstellt...",
+        "creating-file": "Konfiguration %m/%f existiert aktuell nicht, wird aber gleich erstellt...",
```

Comment:
These changes won't be live to SCNX, please use weblate to change translations

Reply:
I dont want them to be live, I just want them fixed locally xD